py/examples/device_tapdetector_example/README.md

```diff
@@ -0,0 +1,31 @@
+## Tap Detector
+### About
```

Comment:
I think the structure of the headers can benefit from some reorganization. I'm thinking something like
- About
- Usage
  - Example (for sample data)
  - understanding CLI options
    - use -h from CLI
  - Additional notes on data compatibility
    - Add notes here about having 2 files with 3 data streams etc.  
- Re-create experiment (needs a better name)
  - start recording session
  - tap
  - end recording session
  - use tap detector

You can choose different wording, but I'm trying to make the point that a good document structure can add a lot of clarity.

Reply:
I will reorder the organization, I see how the current method makes it a little unclear since it doesn't read the best from top to bottom.